mappings/net/minecraft/client/network/packet/BlockPlayerActionS2CPacket.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_4463 net/minecraft/client/network/packet/BlockPlayerActionS2CPacket
+	FIELD field_20319 action Lnet/minecraft/class_2846$class_2847;
+	FIELD field_20320 logger Lorg/apache/logging/log4j/Logger;
```

Comment:
Isn't this a `static final` field? If yes, it should be `LOGGER`.

Reply:
Right, whoops. Fixed.